docs/make.jl

```diff
@@ -40,8 +39,7 @@ end
 
 # setup examples using Literate.jl
 literate_paths = [
-    Base.Filesystem.joinpath(project_path, "docs/src/tutorial/ps_def.jl"),
-    Base.Filesystem.joinpath(project_path, "docs/src/tutorial/particles.jl"),
+    #    Base.Filesystem.joinpath(project_path, "docs/src/tutorial/particles.jl"),
```

Comment:
Just commenting here so it won't be forgotten

Reply:
fixed in 85c9c4e